Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -65,7 +65,10 @@ import BraintreeCore
     @objc(initWithAPIClient:universalLink:)
     public convenience init(apiClient: BTAPIClient, universalLink: URL) {
         self.init(apiClient: apiClient)
-        self.universalLink = universalLink
+        
+        /// appending a PayPal app switch specific path to verify we are in the correct flow when
+        /// `canHandleReturnURL` is called
+        self.universalLink = universalLink.appendingPathComponent("braintreeAppSwitchVenmo")
```

Comment:
Should we consider the case that a merchant isn't truly "wildcarding" anything after their path and is actually doing a strict path check? If so, this would break their integration

Reply:
We are pretty explicit in our need to wildcard here: https://developer.paypal.com/braintree/docs/guides/paypal/app-switch/ios/v6#set-up-universalapp-links. We already append details like `success/cancel` and query parameters to the URLs so path checks should already account for custom details. What do you think?